cas-server-core/src/main/java/org/jasig/cas/CasVersion.java

```diff
@@ -43,4 +47,27 @@ private CasVersion() {
     public static String getVersion() {
         return CasVersion.class.getPackage().getImplementationVersion();
     }
+
+    /**
+     * Gets last modified date/time for the module.
+     * @return the date/time
+     */
+    public static DateTime getDateTime() {
```

Comment:
It's really useful for our CAS server at Heroku, but I'm a bit skeptical of the usefulness for any regular CAS deployment.